src/test/java/uk/gov/justice/hmpps/prison/service/BookingServiceTest.java

```diff
@@ -486,48 +493,54 @@ public void getBookingVisitsWithVisitor() {
                 .build());
 
         var page = new PageImpl<>(visits);
```

Comment:
messy changes now the implementation is changed. NB the integration tests remain unchanged